app/models/airport.rb

```diff
@@ -1,12 +1,9 @@
 class Airport < ActiveRecord::Base
   has_many :flights
 
-  validates :name, presence: true
-  validates :location, presence: true
+  validates :name, :location, presence: true
 
-  class << self
-    def search(datum)
-      where("location ILIKE ? OR name ILIKE ?", "%#{datum}%", "%#{datum}%")
-    end
+  def self.search(datum)
```

Comment:
**Code quality - critical**

Follow the Ruby Style Guide on how to define class methods.